lib/node-utils/index.ts

```diff
@@ -525,21 +525,27 @@ const matcherNamesHandlePromise = [
 	'rejects',
 	'toResolve',
 	'toReject',
+	'toBeRejected',
+	'toBeRejectedWith',
```

Comment:
Can you add a comment before this line specifying these are coming from jasmine?

Reply:
added in db05d13d673bfa2dd4f5d175c42c3e3f092e2040